scripts/release/rpm/pipeline_azurelinux.sh

```diff
@@ -16,6 +16,7 @@ docker build \
     --target build-env \
     --build-arg cli_version=${CLI_VERSION} \
     --build-arg image=${IMAGE} \
+    --secret id=PIP_INDEX_URL \
```

Comment:
`--build-arg` also works, but I prefer `--secret` as the index url contains credential and secret does not record its value in image.

Ref: https://docs.docker.com/build/building/secrets/

Reply:
It is interesting that `env` can be omitted:

https://docs.docker.com/build/building/secrets/#sources

> When you use secrets from environment variables, you can omit the `env` parameter to bind the secret to a file with the same name as the variable.